be/src/storage/tablet_updates.cpp

```diff
@@ -2075,8 +2075,8 @@ void TabletUpdates::_apply_compaction_commit(const EditVersionInfo& version_info
     auto manager = StorageEngine::instance()->update_manager();
     auto tablet_id = _tablet.tablet_id();
     auto& version = version_info.version;
-    LOG(INFO) << "apply_compaction_commit start tablet:" << tablet_id << " version:" << version_info.version.to_string()
-              << " rowset:" << rowset_id;
+    VLOG(1) << "apply_compaction_commit start tablet:" << tablet_id << " version:" << version_info.version.to_string()
```

Comment:
I think it's best to keep this log and we often refer to it when troubleshooting issues.

Reply:
Ok